cosmo_tester/test_suites/test_simple_manager_blueprint/abstract_single_host_test.py

```diff
@@ -72,8 +72,10 @@ def setup_simple_manager_env(self):
             self.local_env.outputs()['simple_vm_private_ip_address']
 
     def bootstrap_simple_manager_blueprint(self, override_inputs=None):
-        self.manager_blueprints_repo_dir = clone(MANAGER_BLUEPRINTS_REPO_URL,
-                                                 self.workdir)
+        self.manager_blueprints_repo_dir = clone(
+            MANAGER_BLUEPRINTS_REPO_URL, self.workdir,
+            branch=os.environ.get('CLOUDIFY_MANAGER_BLUEPRINTS_BRANCH',
```

Comment:
clone falls back to using `BRANCH_NAME_CORE`, no need for an additional environment variable